src/backend/web/routes/login.js

```diff
@@ -3,8 +3,9 @@ const session = require('express-session');
 const bodyParser = require('body-parser');
 const cookieParser = require('cookie-parser');
 const passport = require('passport');
+require('../../login/usingPassport');
```

Comment:
Why are we `require`'ing `../../login/usingPassport` twice? (line 6 and 7).  Can we consolidate this?

Reply:
Oops - was using this for testing purposes. Fixed.